examples/simple-lambda.hs

```diff
@@ -40,3 +45,53 @@ code = "\
 \  callback(null, \"some success message\"); \
 \ } \
 \ "
+
+
+role :: Resource
+role = 
+  resource "IAMRole" $
+  IAMRoleProperties $
+  iamRole rolePolicyDocumentObject
+  & iamrPolicies ?~ [ executePolicy ]
+  & iamrRoleName ?~ "MyLambdaBasicExecutionRole"
+  & iamrPath ?~ "/"
+
+  where
+    executePolicy = 
+      iamPolicies
+      (HM.fromList
```

Comment:
> heh, that QQ business strikes me as a step back to writing CF template in json :smile:

Haha true, but the policy stuff has to be given in JSON anyway, and at least aeson-qq validates the JSON at compile time. Your solution is perfectly fine too for now.